src/main/java/hello/controller/IssueController.java

```diff
@@ -0,0 +1,277 @@
+
+import java.io.IOException;
+import java.util.*;
+
+@RestController
+public class IssueController {
+
+    @Autowired
+    private FixesRepo fixesRepo;
+    @Autowired
+    private IssueRepo issueRepo;
+
+    private Driver driver;
+
+    private void setNeo4jDriver(String url) {
+        final String username = "neo4j";
+        final String password = "password";
+        driver = GraphDatabase.driver( url, AuthTokens.basic( username, password ) );
+    }
+
+    class PostResponse {
+        private Integer fixId;
+        private String status;
+        public PostResponse(Integer fixId, String status) {
+            this.fixId = fixId;
+            this.status = status;
+        }
+    }
+
+    class GetResponseIssue {
+        private Integer id;
+        private String issueType;
+        private String line;
+        private String file;
+        private String columnName;
+        private Boolean isFixed;
+
+        GetResponseIssue(Integer id, String issueType, String line, String file, String columnName, Boolean isFixed) {
+            this.id = id;
+            this.issueType = issueType;
+            this.line = line;
+            this.file = file;
+            this.columnName = columnName;
+            this.isFixed = isFixed;
+        }
+    }
+
+    class GetResponse {
+        private Integer fixId;
+        private List<GetResponseIssue> issues;
+        private String s3Link;
+        private String status;
+
+        public GetResponse(Integer fixId, String s3Link, String status) {
+            this.fixId = fixId;
+            this.s3Link = s3Link;
+            this.status = status;
+            this.issues = new ArrayList<GetResponseIssue>();
+        }
+
+        public GetResponse(Integer fixId, String status) {
+            this.fixId = fixId;
+            this.status = status;
+            this.s3Link = "";
+            this.issues = new ArrayList<GetResponseIssue>();
+        }
+
+        public List<GetResponseIssue> getIssues() {
+            return issues;
+        }
+    }
+
+    class CodeGenPostResponse {
+        Integer id;
+        CodeGenPostResponse(Integer id) {
+            this.id = id;
+        }
+    }
+
+    class CodeGenPostRequest {
+        String url;
+        String username;
+        String password;
+        Set<Integer> fileIds;
+
+        CodeGenPostRequest(String url, String username, String password) {
+            this.url = url;
+            this.username = username;
+            this.password = password;
+            this.fileIds = new HashSet<Integer>();
+        }
+    }
+
+    class CodeGenGetResponse {
+        String status;
+        String Url;
+
+        CodeGenGetResponse(String status, String Url) {
+            this.status = status;
+            this.Url = Url;
+        }
+    }
+
+    private static final String ALPHA_NUMERIC_STRING = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+
+    public static String randomAlphaNumeric() {
+        int count = 10;
```

Comment:
nitpick: nothing about the method indicates it returns a string of length 10. I recommend you either add that info to the name of the method or take the length as input.